### ISSUE TITLE
Error message SublimeLinter 4.12: cls.syntax / cls.version / cls.defaults

### ISSUE THREAD

**Issue by hirschferkel** [mmaday/SublimeLinter-contrib-htmlhint#12]
Does anyone know how to avoid this error messages in Sublime:

```
reloading plugin SublimeLinter-contrib-htmlhint.linter
htmlhint: Defining 'cls.syntax' has no effect anymore. Use http://www.sublimelinter.com/en/stable/linter_settings.html#selector instead.
htmlhint: Defining 'cls.version_args' has no effect. Please cleanup and remove these settings.
htmlhint: Defining 'cls.version_re' has no effect. Please cleanup and remove these settings.
htmlhint: Defining 'cls.version_requirement' has no effect. Please cleanup and remove these settings.
htmlhint disabled. 'cls.defaults' is mandatory and MUST be a dict.
```

**Comment by jeanleem6** [comment]
Got the same error. 
: Sublime Text Build 3207
: mac os high sierra 10.13.6
Hope to get the error handling @thedaviddias

**Comment by thedaviddias** [comment]
@jeanleem6 	I'm gone take a look

**Comment by james2doyle** [comment]
You can see the fixes here: https://github.com/borela-forks/SublimeLinter-contrib-htmlhint/commit/32b8cec0e046c89706e308ee3fb8457b615b23fa

**Comment by hirschferkel** [comment]
@james2doyle And how will it be updated? When I try to update the package with Sublime, it says there is no update available?

**Comment by james2doyle** [comment]
I'm not sure what you mean. That link is to a different repository that has implemented the fixes. They would need to be pulled into this project and a new release would need to be tagged

**Comment by hirschferkel** [comment]
Ah I understand. Is anybody out there who will do this and should there not be at least a pull request being sent? I can't see one, so maybe now one knows about the fix from the ones who should merge it...

Best, hirschferkel

**Comment by graylogo** [comment]
the same error.
what should we do?

**Comment by graylogo** [comment]
SublimeLinter: ERROR:
=====================
htmlhint: Defining 'cls.syntax' has no effect anymore. 
Use http://www.sublimelinter.com/en/stable/linter_settings.html#selector instead.

SublimeLinter: ERROR:
=====================
htmlhint disabled. 'cls.defaults' is mandatory and MUST be a dict.

what"s mean?  can't solve it .....

**Comment by jeffbyrnes** [comment]
@graylogo per above comments, there are fixes out there in other folks’ forks of this repo, but the maintainer of this project has not pulled them in or implemented them themselves, and thus there is no “official” release of these fixes to PackageControl.

~The quickest solution for you would be to `git clone` one of the forks in your Sublime Text Packages directory, which would override the installed version. Then, once the fixes are released officially, you can delete the clone.~

See my comment below.

**Comment by jeffbyrnes** [comment]
@thedaviddias, any chance of releasing the fixes that are awaiting on master, but not yet released? I refer to these: https://github.com/htmlhint/SublimeLinter-contrib-htmlhint/compare/1.1.4...master

@graylogo you don’t even need to clone a fork of this; I just noticed that the master branch of this repo has the fixes needed.

**Comment by graylogo** [comment]
> @graylogo per above comments, there are fixes out there in other folks’ forks of this repo, but the maintainer of this project has not pulled them in or implemented them themselves, and thus there is no “official” release of these fixes to PackageControl.
> 
> The quickest solution for you would be to `git clone` one of the forks in your Sublime Text Packages directory, which would override the installed version. Then, once the fixes are released officially, you can delete the clone.

thank you! I will try it.

**Comment by mmaday** [comment]
Fixed in the [2.0.1](https://github.com/htmlhint/SublimeLinter-contrib-htmlhint/releases/tag/2.0.1) release.
